services/dask-sidecar/src/simcore_service_dask_sidecar/computational_sidecar/docker_utils.py

```diff
@@ -260,8 +260,9 @@ async def _parse_container_docker_logs(
                         log_line,
                     )
                     await log_fp.write(log_line.encode("utf-8"))
+                    # NOTE: here we remove the timestamp, only needed for the file
                     await _parse_and_publish_logs(
-                        log_line,
+                        log_line.split(" ", maxsplit=1)[1],
```

Comment:
No need to add `" "`, spaces are a default. SEE
```cmd
>>> "0, 1  2 3 4".split(maxsplit=1)
['0,', '1  2 3 4']
>>> "0, 1  2 3 4".split(maxsplit=1)[1]
'1  2 3 4'
```

BUT what happens if `log_line` does not have 2 elements? 
```
>>> "0".split(maxsplit=1)[1]
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
IndexError: list index out of range
```

Reply:
why would it? now that timestamps are in the logs from docker. these are always there.
I prefer to add ```" "``` it's more explicit